### PR TITLE
Add filesystem connector docs

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

Add filesystem connector docs.

## Brief change log

  - Add filesystem connector docs.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: docs/content/connectors/overview.md [hunk 1]
```diff
@@ -26,10 +26,11 @@ where "Y" means supported, "Y/N" means partially supported, "N" means unsupporte
 
 ### Connectors and Modes Supported by FlinkProcessor
```
Comment (Sxnan) [inline docs/content/connectors/overview.md:27]: Can you add similar tables for `LocalProcessor` and `SparkProcessor`>

File: docs/content/connectors/filesystem.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Filesystem
+
+FeatHub provides `FileSystemSource` to read data from file systems and `FileSystemSink`
+to materialize feature view to file systems.
+
+## Supported Processors and Modes
+
+- Local: Batch Scan, Batch Append
+- Flink: Batch Scan, Batch Append
```
Comment (Sxnan) [inline docs/content/connectors/filesystem.md:9]: `Batch` and `Streaming` refer to the execution mode and Flink currently only supports streaming mode. Therefore, Flink should support Streaming Scan and Streaming Append. Note that Streaming Scan implies both bounded and unbounded.

**Review by Sxnan — COMMENTED** [review]
Thanks for your PR. I left some comments below.

**Comment by jiangxin369** [comment]
@Sxnan Could you help review again?

**Comment by Sxnan** [comment]
Thanks for the update! LGTM. @lindong28 Could you take a look at this PR?

**Comment by lindong28** [comment]
Thanks for the PR. LGTM.

**Pull request merged**